earthaccess/auth.py

```diff
@@ -210,8 +235,14 @@ def _netrc(self) -> bool:
         return authenticated
 
     def _environment(self) -> bool:
-        username = os.getenv("EDL_USERNAME")
-        password = os.getenv("EDL_PASSWORD")
+        if "EDL_USERNAME" in os.environ:
+            username = os.getenv("EDL_USERNAME")
+        if "EARTHDATA_USERNAME" in os.environ:
+            username = os.getenv("EARTHDATA_USERNAME")
+        if "EDL_PASSWORD" in os.environ:
+            password = os.getenv("EDL_PASSWORD")
+        if "EARTHDATA_PASSWORD" in os.environ:
+            password = os.getenv("EARTHDATA_PASSWORD")
         authenticated = self._get_credentials(username, password)
         if authenticated:
             print("Using environment variables for EDL")
```

Comment:
Recommend we update line 251 to EARTHDATA_USERNAME and EARTHDATA_PASSWORD, if that's the convention we'd like to encourage.